File: app/src/main/java/com/example/hytool/total_directory/dagger/component/CommonActivityComponent.java
package com.example.hytool.total_directory.dagger.component;


import com.example.hytool.total_directory.base.BaseActivity;
import com.example.hytool.total_directory.dagger.moudle.CommonActivityModule;

import dagger.Component;

@Component( modules = CommonActivityModule.class)
public interface CommonActivityComponent {
    void inject(BaseActivity mLoginActivity);
}

File: app/src/main/java/com/hyy/hytool/dagger/component/CommonActivityComponent.kt
package com.hyy.hytool.dagger.component

import com.hyy.hytool.base.BaseActivity
import com.hyy.hytool.dagger.noudle.CommonActivityModule
import dagger.Component

/**
 * @Author : HYY
 * 描述:
 * 时间:      2022/05/2022/5/31
 * 软件包名:   com.hyy.hytool.dagger.component
 */
@Component(modules = [CommonActivityModule::class])
interface CommonActivityComponent {
    fun inject(mLoginActivity: BaseActivity?)
}